api_tests/src/post.spec.ts

```diff
@@ -710,3 +711,25 @@ test("Fetch post via redirect", async () => {
   expect(gammaPost.post?.post.ap_id).toBe(alphaPost.post_view.post.ap_id);
   await unfollowRemotes(alpha);
 });
+
+test("Block post that contains banned URL", async () => {
+  let editSiteForm: EditSite = {
+    blocked_urls: ["https://example.com/"],
+  };
+
+  await epsilon.editSite(editSiteForm);
+
+  await delay(5_000);
+
+  if (!betaCommunity) {
+    throw "Missing beta community";
+  }
+
+  expect(createPost(epsilon, betaCommunity.community.id)).rejects.toStrictEqual(
+    Error("blocked_url"),
+  );
+
+  // Later tests need this to be empty
+  editSiteForm.blocked_urls = [];
+  await epsilon.editSite(editSiteForm);
```

Comment:
I think the random test failures happen because the blocklist is still being cached for a few seconds after this. I would fix it by blocking a different domain such as `https://evil.com`, and passing an explicit link to `createPost()`.

Reply:
Tests seem to be passing now, so it was probably just an issue with my runner.

Reply:
Or its due to the url blocklist and will fail again later. Better to resolve it now.

Reply:
I changed the URL, but I'm still confused why "Sticky a post" was failing given it doesn't use `epsilon`.